TombEngine/Objects/TR5/Entity/tr5_twogun.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+class EulerAngles;
+class GameVector;
+struct ItemInfo;
+
+//temp
+class Pose;
+class Vector3i;
+
+namespace TEN::Entities::Creatures::TR5
+{
+	void InitialiseHeavyGuard(short itemNumber);
+	void HeavyGuardControl(short itemNumber);
+
+	void HeavyGuardHit(ItemInfo& target, ItemInfo& source, std::optional<GameVector> pos, int damage, bool isExplosive, int jointIndex);
+	//void FireHeavyGuardRaygun(ItemInfo& item, bool fireRight, bool spawnLaser);
```

Comment:
What are these two commented out functions supposed to be doing?

Reply:
I'm still working on this entity.